Add the alica designer runtime

veerajsk opened the pull request:
Add the necessary jar files & docker-compose file required to run the
web based plan designer & the code generation

abhi1287 commented:
Might not be a good idea to put this under version control. We can host it on drive and give a link in alica repo readme or distribute the zip along with official release artifacts.

veerajsk commented:
> Might not be a good idea to put this under version control. We can host it on drive and give a link in alica repo readme or distribute the zip along with official release artifacts.

@abhi1287 shall I use git-lfs? That way users will only need to clone this repo to get whatever they need & the release process becomes simpler - just need to tag a commit.

veerajsk committed:
Add the alica designer runtime

Add the necessary jar files & docker-compose file required to run the
web based plan designer & the code generation

abhi1287 commented on alica_designer_runtime/README.md:
```diff
@@ -0,0 +1,12 @@
+- Create/Modify the plans & export them to the your application's etc folder
+- To generate the source code you need to download a couple of jar files:
+    1. PlanDesignerFX-Codegeneration-0.1.1.1328.jar from https://drive.google.com/file/d/158mziFym4RlPp2COBuIpzb0m90abWmvg/view?usp=sharing & place it under the alica_supplementary/alica_designer_runtime/codegen folder
+    2. alica-plan-designer-fx-default-plugin-0.1.1-SNAPSHOT.1.jar from https://drive.google.com/file/d/1Og7RLLKany6T2TKUYfUIaVVXZWfOtlJB/view?usp=sharing & place it under the alica_supplementary/alica_designer_runtime/codegen/plugins folder
```
we can zip and keep the codegen folder on drive and ask users to download and unzip in this directory.

veerajsk replied:
done

veerajsk committed:
Address review comment
